pkgs/data/themes/base16-schemes/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "base16-schemes";
-  version = "unstable-2022-12-16";
+  version = "unstable-2023-5-2";
```

Comment:
I suggest to pad month and day with a 0 so the version string can be lexicographically sorted while keeping the date in the right order:

```
"unstable-2023-05-02"
```

Reply:
It seems like the 0s have disappeared from your last commit.